migreringer/src/main/resources/db/migration/defaultDS/4.1/V4.1_31__TFP-5732-medlemskapsvurdering.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE MEDLEMSKAPSVILKÅR_VURDERING
+(
+    ID                 NUMBER(19)                              not null
+        constraint PK_MEDLEMSKAPSVILKÅR_VURDERING primary key,
+    VILKAR_RESULTAT_ID NUMBER(19)                              not null
+        constraint FK_MEDLEMSKAPSVILKÅR_VURDERING_1 references VILKAR_RESULTAT,
+    OPPHOR_FOM         DATE,
+    OPPHOR_ARSAK       VARCHAR2(100 char) default '-'          not null,
```

Comment:
Hva med bare ARSAK siden den skal settes for opphørFom eller medlemFom (en av dem må være satt)?
Blir det for hacky med bare arsak, fom (+vilkar_type) ?

Reply:
Tenkte at vi ikke trenger årsak for forutgående, ettersom denne lagrer direkte på vilkåret